ndarbas06/tobuloRadimas.php

```diff
@@ -0,0 +1,38 @@
+<?php
+    function dalikliuRadimas($s) {
+        $dal[] = NULL;
```

Comment:
Gal norejote parasyti `$dal = []` ? Tuomet neliktu to vieno if'o kur zemiau yra.